grpc_api/descriptor_record_store.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
```

Comment:
Shall we rename this file to `grpc_api.proto`

Reply:
This was renamed to match `DESCRIPTOR_RECORD_STORE_DESC` constant in [ballerina file](https://github.com/ballerina-guides/ballerina-in-action-samples/pull/1/files#diff-fd21bb0c8db4c66516b982c5825527b61d511ccaff252095c8e7b0d6a1730dc9R12)

If we change this `grpc_api.proto` the constant is changing and we need to use `GRPC_API_DESC`